boot/boot.go

```diff
@@ -150,7 +144,7 @@ type bootState interface {
 	// revisions retrieves the revisions of the current snap and
 	// the try snap (only the latter might not be set), and
 	// whether the snap is in "trying" state.
-	revisions() (snap, try_snap *NameAndRevision, trying bool, err error)
```

Comment:
Hm, I wonder why the static checks have not warned about those. In any case, thanks for the fix!

Reply:
well tbh the thing that is complaining for me is go-lint, which is more strict than go vet, so run-checks won't complain about it. just a little nit pick thing that I don't think we need to block PR's on, so we don't need to add anything to run-checks